TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -960,7 +1082,12 @@ void TTSSpeaker::GStreamerThreadFunc(void *ctx) {
         if(!speaker->m_flushed) {
             speaker->speakText(*data.client->configuration(), data);
         }
-
+        
+        if(!speaker->m_flushed && speaker->m_remoteError)
+        {
```

Comment:
So, my understanding is we fall back to local endpoint on the very first network failure observed... So even the spurious failures 500 responses we have seen would trigger using the local endpoint configured